Makefile

```diff
@@ -73,16 +73,22 @@ GO ?= CGO_ENABLED=0 go
 # Allow setting of go build flags from the command line.
 GOFLAGS := 
 
-# Location to install dependencies to
-GOBIN ?= $(GOPATH)/bin
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
```

Comment:
did the previous version (using `?=`) not work?

Reply:
It didn't work in the Github action